spec.md

```diff
@@ -215,6 +215,30 @@ help intermediate gateways determine how to route the events.
     * urn:event:from:myapi/resourse/123
     * mailto:cncf-wg-serverless@lists.cncf.io
 
+### subject
+* Type: `String`
+* Description: This describes the subject of the event in the context of the
+  event producer (identified by `source`). In publish-subscribe scenarios,
+  a subscriber will typically subscribe to events emitted by a `source`,
+  but the `source` identifier alone might not be sufficient as a qualifier for
+  any specific event if the `source` context has internal sub-structure.
+* Example: A subscriber might register interest for when new blobs are created
+  inside a blob-storage container. In this case, the event `source` identifies
+  the subscription scope (storage container), the `type` identifies the "blob
+  created" event, and the `id` uniquely identifies the event instance to
+  distinguish separate occurrences of a same-named blog having been created;
+  the name of the newly created blob is carried in `subject`.
+
+  Identifying the subject of the event in context metadata (opposed to only in
```

Comment:
For source, we have these examples:

> * Examples
    * https://github.com/cloudevents/spec/pull/123
    * /cloudevents/spec/pull/123
    * urn:event:from:myapi/resourse/123
    * mailto:cncf-wg-serverless@lists.cncf.io

Would using subject turn them into:

    * source: https://github.com/cloudevents/spec/
    * subject: pull/123

    * source: /cloudevents/spec/pull/123
    * subject: pull/123

    * source: urn:event:from:myapi/resourse/123
    * subject: UPDATE `<-- ? picked a verb`

    * source: mailto:cncf-wg-serverless@lists.cncf.io
    * subject: Help

Reply:
The last example bring the point home rather nicely. That's exactly what subject is.